api/spec/src/entitlements/entitlements.tsp

```diff
@@ -0,0 +1,321 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+// TODO: does this have to be in a separate namespace?
+namespace OpenMeter.Entitlements;
+
+/**
+ * Order by options for entitlements.
+ */
+@friendlyName("EntitlementOrderBy")
+enum EntitlementOrderBy {
+  #suppress "@openmeter/api-spec/casing" "Use existing values"
+  CreatedAt: "createdAt",
+  #suppress "@openmeter/api-spec/casing" "Use existing values"
+  UpdatedAt: "updatedAt",
+}
+
+@route("/api/v1/entitlements")
+@tag("Entitlements (Experimental)")
+interface Entitlements {
+  /**
+   * List all entitlements for all the subjects and features. This endpoint is intended for administrative purposes only.
+   * To fetch the entitlements of a specific subject please use the /api/v1/subjects/{subjectKeyOrID}/entitlements endpoint.
+   * If page is provided that takes precedence and the paginated response is returned.
+   */
+  @get
+  @operationId("listEntitlements")
+  @summary("List all entitlements")
+  list(
+    /**
+     * Filtering by multiple features.
+     *
+     * Usage: `?feature=feature-1&feature=feature-2`
```

Comment:
Nit: is this actually used somewhere? Example generation could take care of this in documentation.

Reply:
I just copied the current docs everywhere